### ISSUE TITLE
[ENH] Normaliser/StandardScaler Transformer

### ISSUE THREAD

**Issue by TonyBagnall** [aeon-toolkit/aeon#605]
### Describe the feature or idea you want to propose

we currently normalise series prior to clustering using scikit StandardScaler. However, this only works with 2D numpy and requires transposing series (since we want to normalise over a single series not over a single time point. This is messy to do with 3D numpy and easy to get wrong. Normalising along the wrong axis will completely skew results. I'm not a fan of wrapping scikit code, but in this instance I think it is worthwhile to do

### Describe your proposed solution

I think we need a CollectionTransformer that can normalise each series across each channel independently.  Perhaps just call it StandardScaler to avoid normalise/normalize arguments :)

**Comment by aiwalter** [comment]
We can already do that with the `FromTabularToSeriesAdopter(StandardScaler())`

**Comment by TonyBagnall** [comment]
found it, TabularToSeriesAdaptor, I'll give it a go, thanks.

**Comment by TonyBagnall** [comment]
just tried it, works correctly, but this is what I think is going on internally. It converts the data into a pd.Multiindex, creates n_cases*n_channels versions of the standard scaler, splits the mulitindex into n_cases different dataframes, converts the dataframes back into numpy arrays before finally calling StandardScaler. 

This is very inefficient


```python
for i in range(100,1000,100):
    X = np.random.random(size=(i,4,1000))
    sc = StandardScaler()
    tsa = TabularToSeriesAdaptor(sc)
    start = time.time()
    x2 = tsa.fit_transform(X)
    slow = time.time()-start

    start = time.time()
    for x in X:
        x1 = np.transpose(x)
        norm = sc.fit_transform(x1)
        x1 = np.transpose(norm)
    fast=time.time() - start
    print(i,",",fast,",",slow,",", slow/fast)
```
it gets worse as the number of cases  increases. This is the plot of the ratio of slow (TabularToSeriesAdaptor) and fast (code above), suggesting that it is in fact a class of inefficiency worse, god knows how! For a dataset (1700,4,1000) (hardly very big) it takes 45 seconds as opposed to 0.59 seconds for native code. Obviously will be some over head for a class, but this is not acceptable to me, I think I will write a simpler transformer for this task

![image](https://github.com/aeon-toolkit/aeon/assets/9594042/abd8b980-0dfb-4b88-8169-e759b5e7f1a3)
